bluebottle/projects/models.py

```diff
@@ -515,6 +515,18 @@ def get_tweet(self, **kwargs):
     class Meta(BaseProject.Meta):
         ordering = ['title']
 
+    class Analytics:
+        type = 'project'
+        tags = {
+            'sub_type': 'project_type',
+            'status': 'status.name',
+            'status_slug': 'status.slug',
+            'theme': 'theme.name',
+            'theme_slug': 'theme.slug',
+            'location': 'location.name',
+            'country': 'location.country.name'
```

Comment:
I will update the country v. location.

The analytics part catches attribute errors so no location.name is ok.